core/templates/dev/head/domain/utilities/UrlInterpolationService.js

```diff
@@ -119,7 +119,8 @@ oppia.factory('UrlInterpolationService', [
         }
 
         var escapedInterpolationValues = {};
-        for (var varName in interpolationValues) {
+        var varName;
+        for (varName in interpolationValues) {
```

Comment:
Instead of declaring a dummy variable varName above can we just use a different name in the while loop below? Maybe something like currentVarName?

Reply:
Done.